main.js

```diff
@@ -0,0 +1,27 @@
+var saveButton = document.querySelector(".save-btn");
+var titleText = document.querySelector("#title");
+var bodyText = document.querySelector("#body");
+var outputContainer = document.querySelector(".output-container");
+
+
+var currentIdeas = {
+    ideas: []
+};
+
+saveButton.addEventListener("click", function() {
+    saveIdea();
+})
+
+function createIdea(){
+    var idea = {
+        title: titleText.value, 
+        body: bodyText.value, 
+        id: Date.now()
+    };
+    return idea;
+}
+
+function saveIdea() {
+    currentIdeas.ideas.push(createIdea());
+    console.log(currentIdeas.ideas[0]);
```

Comment:
```suggestion
    console.log(currentIdeas.ideas);
```
After the convo between the three of us, it seems like we don't need the 0 index, because we only push one new idea at a time, and we should be able to just `console.log(currentIdeas.ideas);`